.github/workflows/release.yaml

```diff
@@ -62,12 +61,9 @@ jobs:
           username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
           password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
 
-      - name: load vm kernel
-        run: |
-          docker pull --quiet ${{ env.VM_KERNEL_IMAGE }}:${{ env.VM_KERNEL_VERSION }}
-          ID=$(docker create ${{ env.VM_KERNEL_IMAGE }}:${{ env.VM_KERNEL_VERSION }} true)
-          docker cp ${ID}:/vmlinuz neonvm/hack/vmlinuz
-          docker rm -f ${ID}
+      - name: build vm kernel
+        run:
+          make kernel
```

Comment:
I would prefer to call the `vm-image` workflow from the release workflow (this requires adding `workflow_call` to `vm-image`). Then we have vm-kernel image as an artifact (published to docker hub, we can even add a custom release tag to the image). 
What do you think? (I'm happy to make all workflows modifications for this)

Reply:
> What do you think?

Yeah, that sounds like a good idea :)

+1 to adding a custom release tag, so that we don't accidentally end up with another image sneaking in.

> I'm happy to make all workflows modifications for this

that'd be wonderful, thanks :)

Reply:
Experimenting in this PR https://github.com/neondatabase/autoscaling/pull/679.